subsys/net/ip/Kconfig.stats

```diff
@@ -99,4 +99,14 @@ config NET_STATISTICS_ETHERNET
 	  requires support from the ethernet driver. The driver needs
 	  to collect the statistics.
 
+config NET_STATISTICS_ETHERNET_VENDOR
+	bool "Vendor specific Ethernet statistics"
+	depends on NET_STATISTICS_ETHERNET
+	default n
```

Comment:
Just noticed this "default n", it can be removed as the bool value is "n" by default. There has been some patches lately that remove the extra defaults so we should not introduce new ones here.

Reply:
Updated the patch.